expirationd.lua

```diff
@@ -353,9 +354,27 @@ end
 --     * force                 -- Run task even on replica.
 --  }
 local function expirationd_run_task(name, space_id, is_tuple_expired, options)
-    if name == nil then
-        error("task name is nil")
-    end
+    checks('string', 'number|string', 'function', {
+        process_expired_tuple = '?function',
+        index = '?number|string',
+        iterator_type = '?string',
+        start_key = '?',
+        tuples_per_iteration = '?number',
+        atomic_iteration = '?boolean',
+        process_while = '?function',
+        iterate_with = '?function',
+        on_full_scan_start = '?function',
+        on_full_scan_complete = '?function',
+        on_full_scan_success = '?function',
+        on_full_scan_error = '?function',
+        args = '?table',
+        full_scan_time = '?number',
+        iteration_delay = '?number',
+        full_scan_delay = '?number',
```

Comment:
I don't know is it relevant or not but maybe FFI numbers should be also available for numeric fields.

```
tarantool> type(1ULL)
---
- cdata
...

tarantool> type(1LL)
---
- cdata
...

tarantool> 1ULL == 1
---
- true
...
```

Reply:
Good point.

Added `cdata` for these arguments in `expirationd.start()`: full_scan_time, iteration_delay, full_scan_delay, tuples_per_iteration, vinyl_assumed_space_len, vinyl_assumed_space_len_factor.

Reply:
NB: There are [checks](https://github.com/tarantool/checks/blob/eb49c159ac1f81d37420454c6574cc003aa8569d/checks.lua#L194-L227) for cdata numbers.